Sources/XiEditor/EditView.swift

```diff
@@ -274,7 +274,7 @@ final class EditView: NSView, NSTextInputClient, TextPlaneDelegate {
         var replacementRange = aRange
         var len = 0
         if let attrStr = aString as? NSAttributedString {
-            len = attrStr.string.count
+            len = attrStr.string.utf16.count
```

Comment:
`string.length` is equal to `string.utf16.count`, so maybe we use that so it's clear that these two branches are doing the same thing?

Reply:
Ah okay I didn't know that the `string` member on `NSAttributedString` was a `String` and not an `NSString`. 😕

This looks good, thanks!